moped-editor/src/components/DataGridPro/DataGridActions.js

```diff
@@ -79,6 +81,10 @@ const DataGridActions = ({
       />,
     ];
   }
+  const DeleteIcon = deleteIcon || (
```

Comment:
This gives us the ability to insert an optional custom icon for the delete grid action.